### DISCUSSION TITLE
Suggestions for Razor files miss the closing html tag

### DISCUSSION THREAD

**Discussion by CamiloTerevinto** [community/community#74783] [Copilot Conversations]
### Select Topic Area

Bug

### Body

As the title says, whenever Copilot suggests code within a Razor file (.cshtml), the closing tag for the initial HTML element is missing from the suggestion, forcing me to manually add it every time.

See example:
![image](https://github.com/community/community/assets/6320854/683c7580-92e1-4b41-a4da-765ace3d76a9)

This happens on the latest version of Visual Studio and Visual Studio Code.

**Comment by ayim** [comment]
Tracking this issue here for Visual Studio: https://developercommunity.visualstudio.com/t/Suggestions-for-Razor-files-miss-the-clo/10535375
